sixtools/info_functions.py

```diff
@@ -0,0 +1,94 @@
+
+
+###scan info utilities###
+def scan_info(scan_id,source='all'):
+    ''' 
+    Prints to the command line, in a human readable way, the header and/or baseline info for the scan defined by scan_id.
+            
+    Parameters
+    ----------
+    scan_id : integer
+        The scan_id, or the location form latest scan_id using -1,-2......, to print data from
+    source : string, optional
+        The source to display info from: can be 'all', 'header' (for header info) or 'baseline' (for baseline info). is 'All' by default.
+        
+    '''
+
+    if source in ['all','baseline','header']:
```

Comment:
There are a few places where it doesn't follow the standard layout. For instance here there should be a space after each comma. @bisogni can I ask you to run this through flake8 and fix these errors, it would be a good learning oportunity? (I am very aware that this is stuff I wrote some time ago and it is wrong, ask @danielballan or @mrakitin about how often they have had to correct similar issues in my code !-) ).

Reply:
@awalter-bnl I think that all the other information that prints out for our use case is not helpful.  I can add an option for `verbose = False` to automatically pass on lists for now if you feel like it is important to let some access this information in an easy way, but I don't want to see it myself as it is not helpful.  

Perhaps, in the future, easily recognized dictionary keys for this excess information can be configured so that it is easy to print or pass these things.  See also https://github.com/NSLS-II/ophyd/issues/587 and https://github.com/NSLS-II/ophyd/issues/586